pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -51,6 +51,9 @@ stdenv.mkDerivation (finalAttrs: {
   postPatch = ''
     # Requires networking.
     sed -i '/check_PROGRAMS += test-crbug1004254/d' test/Makefile.am
+
+    # Test causes error without patch shebangs.
+    patchShebangs doc
```

Comment:
Generally, we prefer specifying individual files when not too much trouble, since it forces us to review the scripts on updates. This is because sometimes a script requires host shebang, e.g. when test scripts is also used in installed tests. Now, this is pretty unlikely in `doc` directory but it is still a good idea IMO, for peace of mind.